### PR TITLE
Removes all mood buffs and debuffs from ERP content

### PR DESCRIPTION
**Pull request by dawsonkeyes**
## About The Pull Request

See title. The moodlets remain but their effective values are 0, making them neutral flavor text rather than giving you superspeed or whatever.

## How This Contributes To The Skyrat Roleplay Experience

Mood buffs affect mood. Mood affects speed. I kid you not, bimbo was a buff of 20. You ever read The Flash?

## Changelog

:cl:
del: No more ERP mood mechanic changes
/:cl:

### CODE REVIEW

**Comment by Arctaisia** [comment]
I can't wait to go at mach 4 because someone slapped my ass

**Comment by dawsonkeyes** [comment]
failed checks are because of your stupid cryo stuff AUGUH

**Comment by SpaceVampire** [comment]
Based PR, you shouldn't be getting free mood buffs by taking fetish quirks that don't require any points.

**Comment by SarmentiCampbell** [comment]
As the only usages for the quirk was their mood buffs, It would be smarter to just remove them at whole

**Comment by softcerv** [comment]
I'd say just limit them to like 1/-1, so they are still there but they basically are next to useless.

**Comment by dawsonkeyes** [comment]
> I'd say just limit them to like 1/-1, so they are still there but they basically are next to useless.

But why? Why would I leave in any buff/debuff?

**Comment by dawsonkeyes** [comment]
> As the only usages for the quirk was their mood buffs, It would be smarter to just remove them at whole

Making them neutral moodlets still leaves 'em as flavor, /shrug

File: modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm [hunk 1]
```diff
@@ -345,17 +345,17 @@
 
 /datum/mood_event/orgasm
 	description = span_purple("Woah... This pleasant tiredness... I love it.\n")
-	mood_change = 8 //yes, +8. Well fed buff gives same amount. This is Fair (tm).
+	mood_change = 0 //8 is not fair(tm)
```
Comment (tf-4) [inline modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm]:
```suggestion
```
Parent value is 0. Apply to all.

Obligatory based committing from master comment

Comment (dawsonkeyes) [reply modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm]: good ideay

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm [hunk 2]
```diff
@@ -430,7 +430,7 @@ But i keeped it as unobtainable breain trauma, so admins can add it through VV *
 //Mood boost
 /datum/mood_event/bimbo
 	description = span_purple("So-o... Help..less... Lo-ve it!\n")
-	mood_change = 20
+	mood_change = 0 //i deserve the right to keep this here even if the parent is also 0 just to say that the psychopath that coded it is jackshit insane for making it +20 previously
```
Comment (tf-4) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm]:
```suggestion
```

Comment (dawsonkeyes) [reply modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm]: i reserve the right to keep it

Comment (tf-4) [reply modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm]: No you don't

Comment (dawsonkeyes) [reply modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm]: :(

**Comment by Acer202** [comment]
> But why? Why would I leave in any buff/debuff?

Characters that spent time doing the do will literally only experience downsides from it. Like, starving, for example, since scenes take a long time for most people to get through. Everyone's characters would leave the bedroom depressed and ready to die because they're starving with nothing to balance it. (Which is kind of funny but it ignores that a lot of people here spend their time doing scenes.) Someone with the unstable quirk could spend all day indulging in their desires and immediately leave being the most depressed person in the universe forever because of mood mechanics.

Are people powergaming this enough to warrant it? I don't see the need to be rid of it _completely_. Bump the numbers down a bit if it's a true balance problem, yeah, but removing it entirely doesn't make sense if you're going to go insane from hunger in cases where these things could balance it out previously. Beating an arcade game probably shouldn't mechanically satisfy someone more than scratching an itch, so to speak.

I think mood as a mechanic in general is kind of crummy, in any case.

**Comment by Lamella-0587** [comment]
please. stop erasing our code. or delete my update entirely, fuck 8 months of my life, c'mon

**Comment by tf-4** [comment]
> 
> 
> please. stop erasing our code. or delete my update entirely, fuck 8 months of my life, c'mon

Feel free to PR it back with better balance in mind. Don't take it so personally.

**Comment by Lamella-0587** [comment]
why I shouldn't take it personally? we're maintaining the code and fixing its bugs instead of maintainers because everybody is busy, we're trying to implement new features while other developers erasing our code and adding old low-quality kinkmate items that copy ours and look ugly as fuck after I removed them twice.

**Comment by nikothedude** [comment]
> why I shouldn't take it personally? we're maintaining the code and fixing its bugs instead of maintainers because everybody is busy, we're trying to implement new features while other developers erasing our code and adding old low-quality kinkmate items that copy ours and look ugly as fuck after I removed them twice.

As I said in my other PR, this is not erasing your work. This is game design. Erasing your work would be literally hacking into your github and deleting your repo, as well as all your local copies.

This argument can't even really be applied here, because, honestly-what is being removed? All that's being changed is the mood bonus given. No features are being gutted.

Nobody here has any kind of vendetta on you, we just see the features you added and we disagree with certain aspects, so we're proposing a change. We aren't even the ones applying it, that'd be the maintainers. It's up to them to decide it your feature really is flawed. Plus-you agreed to having your code modified and additions/removals made to it when you made a PR on an open-source project. No code is holy, there is only the will of the developer and maintainer. I'm sorry if you feel people are trying to erase your work...  but that just isn't true.

**Comment by Lamella-0587** [comment]
sorry, it kinda broke me. I just unhinged because of that returning of old sprites to the vendor again and that PR was like the last drop. you may be right, I just tired of that

**Comment by dawsonkeyes** [comment]
> > But why? Why would I leave in any buff/debuff?
> 
> Characters that spent time doing the do will literally only experience downsides from it. Like, starving, for example, since scenes take a long time for most people to get through. Everyone's characters would leave the bedroom depressed and ready to die because they're starving with nothing to balance it. (Which is kind of funny but it ignores that a lot of people here spend their time doing scenes.) Someone with the unstable quirk could spend all day indulging in their desires and immediately leave being the most depressed person in the universe forever because of mood mechanics.
> 
> Are people powergaming this enough to warrant it? I don't see the need to be rid of it _completely_. Bump the numbers down a bit if it's a true balance problem, yeah, but removing it entirely doesn't make sense if you're going to go insane from hunger in cases where these things could balance it out previously. Beating an arcade game probably shouldn't mechanically satisfy someone more than scratching an itch, so to speak.
> 
> I think mood as a mechanic in general is kind of crummy, in any case.

This... is not true. Furthermore the level of buffs from some things were utterly absurd- the orgasm moodlet made your mood positive even if you were starving. Bimbo literally made you fucking immune to negative moodlets with a positive statistic of 20 (well fed is 8). And yes, there were cases in which these moodlets were literally only upsides even outside of a dorm room- namely masochism, which gave you a mood boost from being hurt without any other downside.

So fuckem, they're out.

**Review by GoldenAlpharex — APPROVED** [review]
Looks good

**Comment by GoldenAlpharex** [comment]
I wouldn't mind the values being +1 and -1, that being said. But I think it's better to try and remove them first, to see how that goes.

**Pull request merged**